### PR TITLE
rtmidi: fix Linux build and install pkgconfig files

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by cho-m** [comment]
Using HTTPS causes cert issue on Linux: https://github.com/Homebrew/homebrew-core/runs/3258134596

Using HTTP causes audit failure on macOS.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -200,6 +200,8 @@ jobs:
             --env-file <(env | egrep 'HOMEBREW|GITHUB') \
             --workdir /tmp/bottles \
             --pull always \
+            --volume /dev/snd:/dev/snd \
+            --privileged \
```
Comment (SMillerDev) [inline .github/workflows/tests.yml:204]: I'm not sure we want this?

Comment (cho-m) [reply .github/workflows/tests.yml:204]: Not something we want.

Just experimenting to see if there was a sound device available on underlying VM to get past current Linux failure.

May just need to skip test (or expect error message) as ALSA either needs an actual device or some `modprobe` commands to enable driver.

Comment (danielnachun) [reply .github/workflows/tests.yml:204]: I think we'll probably want to skip tests that depend on ALSA for now so we can get everything bottled on Linux.  Hopefully we will eventually figure out a way to get proper sound (and graphics) devices enabled in CI so these tests will work, but until we have a solution I think we have to skip them.

**Review by iMichka — APPROVED** [review]

**Pull request closed without merging**